SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -342,4 +344,6 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	bool bUseDevelopmentAuthenticationFlow;
 	FString DevelopmentAuthenticationToken;
 	FString DevelopmentDeploymentToConnect;
+
+	mutable FOnWorkerTypesChanged OnWorkerTypesChangedDelegate;
```

Comment:
Doesn't seem like we use the mutable keyword anywhere else in the codebase, is it needed here?

Reply:
Questionable. I want to be able to observe the settings changes without having to grab a non-const pointer to the settings.

3 options : 
1. Mutable delegate
2. const_cast<>(this) in method
3. Have the user get a non-const pointer to the setting although the have no intent (and should not) be modifying the settings.

Reply:
mutable is correct if your intent is to be able to change this on a const object (i.e. this var isn't part of the constness contract), and is much preferable to const_cast. 👍